torch/_torch_docs.py

```diff
@@ -4185,6 +4185,15 @@ def merge_dicts(*dicts):
 Returns True if inference mode is currently enabled.
 """.format(**common_args))
 
+add_docstr(torch.is_inference, r"""
+is_inference(input) -> (bool)
+
+Returns True if :attr:`input` is an inference tensor
```

Comment:
I'd suggest adding a link to the note where we explain what's an inference tensor (if we already have one). If not we probably need to explain more here :D

Reply:
How about something like:
```
Returns True if :attr:`input` is an inference tensor.

A non-view tensor is an inference tensor if and only if it was
allocated during inference mode. A view tensor is an inference
tensor if and only if the tensor it is a view of is an inference tensor. <-- From RFC

For details on inference mode please see
`Inference Mode <https://pytorch.org/cppdocs/notes/inference_mode.html>`_.
```

Reply:
LGTM!